oadr2/poll.py

```diff
@@ -121,7 +142,12 @@ def poll_vtn_loop(self):
             except Exception as ex:
                 logging.exception("Error in OADR2 poll thread: %s", ex)
 
-            self._exit.wait(self.vtn_poll_interval)
+            self._exit.wait(
```

Comment:
This belongs to a separate pull request.

Reply:
Removed